README.md

```diff
@@ -2,30 +2,31 @@
 
 ## Learning Goals
 
-- Recognize how multiple `class`es can be used together
+- Recognize how multiple classes can share data
+- Recognize how multiple classes can share behaviors
+- Recognize how classes working together develop dependencies
 
 ## Introduction
 
 Using Object Oriented JavaScript, we can encapsulate closely related data and
-behavior. Using `class`es, properties, methods, getters and setters, we can
-finely tune how our data is accessed and modified. And since everything is
-wrapped up, together, no matter where an `class` instance is needed, it behaves
-in a way we've defined. A single cell in a soup of code.
+behaviors. Using classes, properties, methods, getters and setters, we are
+able to finely tune how our data is accessed and modified. Since everything
+is wrapped up together, no matter where a class instance is needed, it
+behaves in the way we've defined.
```

Comment:
Something about this sentence feels ambiguous. Maybe something along the lines of "Since all of the needed/accessible features are linked/combined/grouped together, no matter where an instance of a class is needed, it behaves the same way/similarly,"